test/router.js

```diff
@@ -985,6 +985,31 @@ describe('Router', function () {
   })
 })
 
+describe('req.matchedRoutes', function() {
```

Comment:
Same as comment above, can you make the formatting through here `function () {`?  As per `standard`.

Reply:
Should be fixed in https://github.com/pillarjs/router/pull/86/files#diff-c6bcc5b8d170df93e5b3223fa590f1f7R1133